### PR TITLE
MM-20655: Replacing link to homepage with link to trial page, adding UTM tags, modifying some text

### PR DESCRIPTION
**Pull request by etoaster**
Replacing link to homepage with link to trial page, adding UTM tags, modifying some text

#### Summary
We want to generate Product Qualified Leads, or PQLs, that represent people who are using the product and have demonstrated interest in enterprise features. We want to include a link to the trial page from within the Edition and License page to make it easy to try out enterprise features. With UTM tags, we can better ascertain interest in the more advanced features.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-20655

#### Related Pull Requests
N/A

#### Screenshots
Screen Shot 2019-11-21 at 12 09 28 PM

### CODE REVIEW

File: components/admin_console/license_settings/license_settings.jsx [hunk 1]
```diff
@@ -175,13 +175,13 @@ export default class LicenseSettings extends React.Component {
             // Note: DO NOT LOCALISE THESE STRINGS. Legally we can not since the license is in English.
             edition = (
                 <p>
-                    {'Mattermost Enterprise Edition. Unlock enterprise features in this software through the purchase of a subscription from '}
+                    {'Mattermost Enterprise Edition. Unlock enterprise features in this software by starting a trial subscription at '}
```
Comment (jespino) [inline components/admin_console/license_settings/license_settings.jsx]: shouldn't it be `Mattermost Team Edition. Unlock enterprise...`?

**Comment by wiersgallak** [comment]
@saturninoabril Would you be able to add?  Eric is our new PM, so not as familiar and we are trying to get in for 5.18.  Would there be a major issue is the E2E test was done at a later time?

File: components/admin_console/license_settings/license_settings.jsx (hunk 1 above)
Comment (wiersgallak) [reply components/admin_console/license_settings/license_settings.jsx]: @etoaster & @jespino Recommend adjusting to make it more clear: "Mattermost Enterprise Edition. A license is required to unlock enterprise features in this software. Start a trial subscription at"

Comment (etoaster) [reply components/admin_console/license_settings/license_settings.jsx]: Changed the language to what @wiersgallak suggested.

**Pull request merged**